assets/queries/k8s/psp_allows_sharing_host_pid/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    document.kind == "PodSecurityPolicy"
```

Comment:
In order to improve our searchKey we need to add the metadata information

metadata := document.metadata